okhttp-android/src/main/kotlin/okhttp3/android/AndroidAsyncDns.kt

```diff
@@ -47,6 +48,7 @@ class AndroidAsyncDns(
 
   override fun query(
     hostname: String,
+    originatingCall: Call?,
```

Comment:
Not sure if I want Call or Request here?

There are implementations of Dns that would benefit from the Request info including tags to make decisions.
Our strategy APIs that are oblivious to that end up needing awkward workarounds that interceptors can't fix.

Reply:
not public API anymore